eng/deploy/azure/README.md

```diff
@@ -0,0 +1,96 @@
+# Azure Deployment
+
+To deploy to Azure, this can be done through Azure Container Instances (ACI),
+with the [Docker Azure
+Integration](https://docs.docker.com/cloud/aci-integration/), or with [Azure
+CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+## Deploy with Docker Azure Integration
+
+- [Log into Azure from
+  Docker](https://docs.docker.com/cloud/aci-integration/#log-into-azure).
+
+- [Create an ACI Docker
+  context](https://docs.docker.com/cloud/aci-integration/#create-an-aci-context)
+
+- Switch to the ACI Context
+
+```Shell
+docker context use myacicontext
+```
+
+- Login to Azure Container Registry
+
+```Shell
+az acr login --name edfimeadowlark
+```
```

Comment:
I was thinking something more like this, except you would need to add the rest of the commands into this same Shell block.

```suggestion
- [Log into Azure from
  Docker](https://docs.docker.com/cloud/aci-integration/#log-into-azure).
- [Create an ACI Docker
  context](https://docs.docker.com/cloud/aci-integration/#create-an-aci-context)
- Switch to the ACI Context and Deploy Meadowlark

   ```Shell
   docker context use myacicontext
   az acr login --name edfimeadowlark
   ````
```